repo_validation/scripts/validate_srs_consistency.ps1

```diff
@@ -111,10 +111,10 @@ function Get-SrsTagsFromMarkdown {
     $srsTags = @()
     
     # Pattern to match SRS tags in markdown: **SRS_MODULE_ID_NUM: [** text **]**
-    $pattern = '\*\*SRS_([A-Z0-9_]+)_(\d{2})_(\d{3}):\s*\[\*\*\s*(.*?)\s*\*\*\]\*\*'
-    
-    $matches = [regex]::Matches($Content, $pattern, [System.Text.RegularExpressions.RegexOptions]::Singleline)
-    
+    $pattern = '\*\*SRS_([A-Z0-9_]+)_(\d{2})_(\d{3}):\s*\[\*\*\s*((?:(?!\*\*\]\*\*).)+?)\s*\*\*\]\*\*'
```

Comment:
ha I'm glad this has some tests, even if you throw that into one of those regex visualizers that is a lot to parse in my head #Resolved